runtime/src/bank.rs

```diff
@@ -1199,6 +1203,7 @@ impl Bank {
             feature_set: new(),
             drop_callback: RwLock::new(OptionalDropCallback(None)),
             freeze_started: AtomicBool::new(fields.hash != Hash::default()),
+            max_tx_ingestion_time: Some(SLOT_MS),
```

Comment:
This should come from genesis config, not the constants. Does it need to be on the bank? Why not just pass it as a parameter to banking_stage?